R/logistic_reg.R

```diff
@@ -98,7 +98,7 @@ translate.logistic_reg <- function(x, engine = x$engine, ...) {
         } else if (quo_get_expr(x$args$mixture) == 1) {
           arg_vals$type <- 6      ## lasso
         } else {
-          rlang::abort("For the LiblineaR engine, mixture must be 0 or 1.")
+          cli::cli_abort(c("x" = "For the LiblineaR engine, {.var mixture} must be 0 or 1."))
```

Comment:
```suggestion
          cli::cli_abort("For the LiblineaR engine, {.var mixture} must be 0 or 1.")
```

`cli_abort()` will automatically prefix the first element of the message with an icon!